metadata-integration/java/datahub-client/build.gradle

```diff
@@ -110,6 +110,9 @@ shadowJar {
   relocate 'org.abego.treelayout', 'datahub.shaded.treelayout'
   relocate 'org.slf4j', 'datahub.shaded.slf4j'
   relocate 'javax.annotation', 'datahub.shaded.javax.annotation'
+  relocate 'com.github.benmanes.caffeine', 'datahub.shaded.com.github.benmanes.caffeine'
```

Comment:
These were added due to the bump in pegasus version [here](https://github.com/linkedin/datahub/pull/4340/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L3).